draft-ietf-sedate-datetime-extended.md

```diff
@@ -198,17 +198,62 @@ Timestamp:
 : This term is used in this document to refer to an unambiguous
   representation of some instant in time.
 
+UTC Offset:
```

Comment:
How about "Difference between a given local time and UTC, usually given in negative or positive hours and minutes"?

Reply:
Sounds good. I used your text and added a short example too.